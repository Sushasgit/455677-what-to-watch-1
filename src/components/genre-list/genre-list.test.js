import React from 'react';
import renderer from 'react-test-renderer';
import GenreList from './genre-list.jsx';

it(`GenreList run correctly`, () => {
  const tree = renderer
    .create(<GenreList
      genres={[]}
      onGenreChange={jest.fn()}
      activeGenre={``}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
