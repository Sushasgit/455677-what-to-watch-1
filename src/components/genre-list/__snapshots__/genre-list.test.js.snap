// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList run correctly 1`] = `
<ul
  className="catalog__genres-list"
/>
`;
